magner/lib/assets/styles/pages/table.css

```diff
@@ -179,3 +182,17 @@
     padding: 0 12px;
   }
 }
+
+
+.filters_drawer .table-page_top_filters {
+  flex-wrap: wrap;
+  flex-direction: column;
+
+  .generic-form_only-fields {
+    flex-direction: column;
+
+    .el-form-item {
+      margin-bottom: 10px;
+    }
+  }
+}
```

Comment:
Todo (not related to the selected lines of code): fix styles.

1. "More filters" button is still added to the filters drawer, and it's possible to open the modal. Also, action buttons should take full width and have margins

<img width="1440" alt="Screenshot 2022-09-16 at 09 34 44" src="https://user-images.githubusercontent.com/29180358/190572478-bcf159f2-252a-4456-8765-02ad8fde365a.png">

2. Actions from `header.actions` and `filters` button are stacked vertically

<img width="1440" alt="Screenshot 2022-09-16 at 09 31 30" src="https://user-images.githubusercontent.com/29180358/190572516-3944c7e5-f3f4-42a8-81f7-c7cc78b3fe61.png">